machine_management/creating_machinesets/creating-machineset-vsphere.adoc

```diff
@@ -0,0 +1,14 @@
+[id="creating-machineset-vsphere"]
+= Creating a MachineSet on vSphere
+include::modules/common-attributes.adoc[]
+:context: creating-machineset-vsphere
+
+You can create a different MachineSet to serve a specific purpose in your {product-title} cluster on VMware vSphere. For example, you might create infrastructure MachineSets and related Machines so that you can move supporting workloads to the new Machines.
+
+toc::[]
```

Comment:
This toc should be above the first paragraph, because right now the TOC is appearing after the first paragraph which isn't right.